plugins/catalog-unprocessed-entities/package.json

```diff
@@ -56,6 +56,9 @@
     "@backstage/core-plugin-api": "workspace:^",
     "@backstage/errors": "workspace:^",
     "@backstage/frontend-plugin-api": "workspace:^",
+    "@backstage/plugin-catalog": "workspace:^",
```

Comment:
You'll want to remove this, it's generally not a pattern you'd follow - importing a frontend plugins into another frontend plugin - as that's what the `*-react` and/or `*-common` packages are for. It doesn't look like you use anything from this as well. Once removed it should also help get the CI green or at least move forward.